ci/installation_scripts/install_for_amd64.sh

```diff
@@ -14,5 +14,5 @@ sudo apt-get --assume-yes install "${EXTERNAL_AMD64_PACKAGES[@]}"
 # Install thin-edge packages
 for PACKAGE in "${RELEASE_PACKAGES[@]}"
 do
-    sudo dpkg -i ./"$PKG_DIR"/"$PACKAGE"_0.*_amd64.deb
+    sudo dpkg -i ./"$PKG_DIR"/"$PACKAGE"_1.*_amd64.deb
```

Comment:
We could probably avoid any reference to the version, e.g. `0` or `1`...

```suggestion
    sudo dpkg -i ./"$PKG_DIR"/"$PACKAGE"_*_amd64.deb
```

Reply:
Now done in both scripts